_posts/2015-04-20-sti-store.md

```diff
@@ -0,0 +1,87 @@
+---
+author:
+  name: Dave Tapley
+  twitter: davetapley
+  gplus:
+  bio: Programmer
+  image:
+excerpt:
+  You have an STI model, and you need to add attributes which only exist for certain sub-classes.
```

Comment:
Update this to be more of a lead in to the post. It shows on the index page and should pique the interest of the reader to click through.